compiler/rustc_typeck/src/check/upvar.rs

```diff
@@ -770,6 +772,7 @@ impl<'a, 'tcx> InferBorrowKind<'a, 'tcx> {
         new_kind: ty::ClosureKind,
         upvar_span: Span,
         var_name: Symbol,
```

Comment:
I'd remove var_name here and in the debug statement just log the Place

Reply:
Done